engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -322,4 +300,39 @@ public void map(BatchSource<InputRow> source, BatchSink<Collection<Object[]>> si
         }
         convertOutputRows(out.rows, sink, size);
     }
+
+    private OutputColumns getOutputColumnsInternal(CreateInput createInput) throws Exception {
+        logger.debug("Getting output columns from server");
+        boolean wasInit = false;
```

Comment:
I realize this is just moved code, but something like initializedLocally seem a little more proper.